huntsman/utils/light_control.py

```diff
@@ -0,0 +1,265 @@
+import qhue
+from os import path
+from qhue import Bridge, QhueException, create_new_username
```

Comment:
You do a full `import qhue` above and use, for example, `qhu.create_new_username` below, so this `from qhue import ...` isn't used. You want one or the other.

Reply:
Fixed